android/brave_java_sources.gni

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2019 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+brave_java_sources = [
+  "../../brave/android/java/org/chromium/chrome/browser/init/BraveBrowserInitializer.java",
```

Comment:
what is this relative to?

Reply:
What do you mean? It's a file to init Brave's configuration and settings

Reply:
it's a relative path to the file, but what is it relative to? It's fine, I think maybe just needs a comment to explain since it's obviously not relative to this file